### ISSUE TITLE
battleroyale circle doesn't cover all areas

### ISSUE THREAD

**Issue by Yogbot-13** [yogstation13/Yogstation#11788]
## Test Merges: 
Not Supplied / None
## Reproduction:
All newly added areas (Mainly medbay) since the battle royale PR aren't affected by the storm
 Submitted by: Bibby (212531662305755137)

**Comment by Lostlarry** [comment]
are we talking about the radstorm or what?

**Comment by adamsong** [comment]
No, the battle royale game mode slowly sets portions of the station to ash storm, to simulate the circle closing you get in battle royale games. This doesn't work for some new areas, and they remain unstormed.
